steely/local_steely.py

```diff
@@ -86,16 +92,41 @@ def load_plugins(pdir, plist=None):
     If plist is None then return all plugins found.
     """
     plugins = []
+    loaded_names = set()
     for filename, path in scan_plugins_dir(pdir):
         if plist is None or filename in plist:
             plugins.append(load_plugin(filename, path))
+            loaded_names.add(filename)
+    # Check for any missed plugins
+    if plist:
+        for filename in plist:
+            if filename not in loaded_names:
+                print("Warning: Failed to find",
+                      "'" + filename + "'", "in dir",
+                      "'" + pdir + "'")
     return plugins
 
+def check_args(args):
+    if not (bool(args.p) ^ args.all_plugins):
+        print("One of -p or --all-plugins must be specified")
```

Comment:
at least one of?

Reply:
It doesn't make sense to have both, does it? "at least" implies that.